internal/cli/projects/apikeys/create.go

```diff
@@ -70,14 +70,18 @@ func CreateBuilder() *cobra.Command {
 		Use:   "create",
 		Short: "Create an organization API key and assign it to your project.",
 		Long: `MongoDB returns the private API key only once. After you run this command, immediately copy, save, and secure both the public and private API keys.
+By default, the specified project's parent organization will be assigned as organization member. You must authenicate with a user account or API key with the Organization User Admin role to set organization roles.
```

Comment:
I think for consistency 
>You must authenticate with a user account or API key with the Organization User Admin role to set organization roles

should go after the required role, (also typo in "authenicate"`)